backend/handlers/handlers.go

```diff
@@ -690,6 +691,43 @@ func (handlers *Handlers) getRatesHandler(_ *http.Request) (interface{}, error)
 	return handlers.backend.RatesUpdater().LatestPrice(), nil
 }
 
+func (handlers *Handlers) getBTCParseExternalAmount(r *http.Request) (interface{}, error) {
+	amount := r.URL.Query().Get("amount")
+
```

Comment:
maybe nice to put `type result struct { Success bool, Amount string }` here with the json tags, as we started doing in other places, so one can see the result at a glance instead of checking each return statement.

Reply:
I kind of like the compactness of not declaring the result struct, but no strong opinions about that. I followed your suggestion